src/cow-react/modules/limitOrders/state/limitOrdersAtom.ts

```diff
@@ -26,7 +26,16 @@ export function getDefaultLimitOrdersState(chainId: SupportedChainId | null): Li
   }
 }
 
-export const limitOrdersAtom = atomWithStorage<LimitOrdersState>('limit-orders-atom', getDefaultLimitOrdersState(null))
+export const limitOrdersAtom = atomWithStorage<LimitOrdersState>(
+  'limit-orders-atom:v1',
```

Comment:
why v1?

Reply:
Oh, it's a very good question.
We should add a version to `localStorage` flag and up it each time when we change the state structure.
Also, we will need migrations for the state structure updates.